backend/api.py

```
@@ -1016,7 +1016,10 @@ async def search_files(search_data: SearchRequest, request: Request, background_
                 'index_summaries': isumm_snap, 'cluster_summaries': csumm_snap,
                 'cluster_map': cmap_snap, 'bm25': bm25_snap
             })
-            return StreamingResponse(agent.stream_chat(search_data.query), media_type="text/event-stream")
+            async def agentic_event_generator():
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `agentic_event_generator`

(ANN202)